include/aws/s3/private/s3_checksums.h

```diff
@@ -48,6 +60,8 @@ struct aws_input_stream *aws_checksum_stream_new(
     struct aws_byte_buf *checksum_output);
 
 /**
+ * TODO: properly support chunked encoding.
```

Comment:
nit: should we doc more here on what "properly" means or create a ticket for it?

Reply:
let's create a ticket for it